compiler/ast/ast_query.nim

```diff
@@ -690,6 +690,16 @@ iterator genericParamsInMacroCall*(macroSym: PSym, call: PNode): (PSym, PNode) =
     if posInCall < call.len:
       yield (genericParam, call[posInCall])
 
+proc endsInNoReturn*(n: PNode): bool =
+  ## checks if expression `n` ends in an unstructured exit (raise, return,
+  ## etc) or calls of noreturn proc. This is meant to be called on an semmed `n`.
```

Comment:
```suggestion
  ## Checks if expression `n` ends in an unstructured exit (raise, return,
  ## etc.) or a call of a noreturn proc. This is meant to be called on a
  ## semmed `n`.
```
Fixes a few small issues (`an` instead of `a`, plural for call, etc.).